source/_docs/guides/launch/05-redirects.md

```diff
@@ -25,53 +25,90 @@ In this lesson, we'll redirect incoming requests to a primary domain name (e.g.,
 
   If you run into issues, please refer to [this documentation](/docs/sftp/#sftp-connection-information).
 
-4. Now open the `code` folder in your SFTP client, and download your site's `settings.php` (Drupal) or `wp-config.php` (WordPress) file.
-5. Edit your configuration file by adding the following snippet for the desired redirect (replace `example.com`):
+4. Open the `code` folder in your SFTP client, and download your site's `settings.php` (Drupal) or `wp-config.php` (WordPress) file.
+5. Edit your configuration file as described for your site's framework:
 
+    <!-- Nav tabs -->
     <ul class="nav nav-tabs" role="tablist">
-      <li role="presentation" class="active"><a href="#www" aria-controls="www" role="tab" data-toggle="tab">Redirect to HTTPS with www</a></li>
-      <li role="presentation"><a href="#non-www" aria-controls="non-www" role="tab" data-toggle="tab">Redirect to HTTPS without WWW</a></li>
+      <li id="wptab" role="presentation" class="active"><a href="#wp" aria-controls="wp" role="tab" data-toggle="tab">WordPress</a></li>
+      <li id="d8tab" role="presentation"><a href="#d8" aria-controls="d8" role="tab" data-toggle="tab">Drupal 8</a></li>
+      <li id="d7tab" role="presentation"><a href="#d7" aria-controls="d7" role="tab" data-toggle="tab">Drupal 7</a></li>
     </ul>
 
     <!-- Tab panes -->
     <div class="tab-content">
-      <div role="tabpanel" class="tab-pane active" id="www">
-      <br>
-      <pre><code class="php hljs">
-    // Require https with www
-    if (isset($_SERVER['PANTHEON_ENVIRONMENT']) &&
-      ($_SERVER['PANTHEON_ENVIRONMENT'] === 'live') &&
-      // Check if Drupal or WordPress is running via command line
-      (php_sapi_name() != "cli")) {
-      if ($_SERVER['HTTP_HOST'] != 'www.example.com' ||
-          !isset($_SERVER['HTTP_X_SSL']) ||
-          $_SERVER['HTTP_X_SSL'] != 'ON' ) {
+    <div role="tabpanel" class="tab-pane active" id="wp">
+    <p markdown="1">Comment out existing code that sets `WP_SITEURL`, such as [Line 79 in Pantheon's WordPress upstream](https://github.com/pantheon-systems/WordPress/blob/master/wp-config.php#L79), otherwise you will get a PHP notice. Then add the following to the end of your `wp-config.php` file (replace `www.example.com`):</p>
```

Comment:
@davidstrauss mentioned that we should put this configuration below the define() for that and simply consume the value from it, but I'm not entirely sure how to accomplish that in this snippet. Any suggestions/tips @ari-gold @lauren-kelly?